app/controllers/broadcasts_controller.rb

```diff
@@ -113,11 +106,14 @@ def possible_recipients
 
   private
 
-    def set_medium_options
-      @medium_options = configatron.to_h[:outgoing_sms_adapter] ? Broadcast::MEDIUM_OPTIONS : Broadcast::MEDIUM_OPTIONS_WITHOUT_SMS
-    end
+  def prep_form_vars
+    @medium_options = configatron.to_h[:outgoing_sms_adapter] ?
```

Comment:
I think so. Configatron might have improved but I remember fighting with it like crazy trying to check for nils. has_key? won't work I think because outgoing_sms_adapter might be nil.